poolakey/src/main/java/ir/cafebazaar/poolakey/callback/GetFeatureConfigCallback.kt

```diff
@@ -0,0 +1,19 @@
+package ir.cafebazaar.poolakey.callback
+
+import android.os.Bundle
+import java.lang.Exception
+
+class GetFeatureConfigCallback {
```

Comment:
I think this must be internal as we check this in our side

Reply:
yes , i think it would be better internal too.